### PR TITLE
Translate the files to Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
- [x] rwd
- [x] tldr
- [x] mixins
- [x] sass
- [x] architecture
- [x] syntax

### CODE REVIEW

**Comment by KittyGiraudel** [comment]
Hello! 👋 

Thank you for bringing the Dutch language for Sass-Guidelines to my attention. Unfortunately, that would also mean translating *the entire content* in Dutch for me to be able to merge this. This pull-request in this state is only translating the user interface (so everything but the actual content). 

Is that something you plan on doing? :)

**Comment by noah-vdv** [comment]
> Is that something you plan on doing? :)

Yeah, just need some time to finish those 👍

**Comment by noah-vdv** [comment]
Alright well I'll add all those `snippets` later 😁

**Comment by KittyGiraudel** [comment]
I see strong progress being done here! I will spend some time on it tomorrow to give it a review. Thank you so much for your work!

File: _locales/nl.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+configuration:
+  version: 1.3
+  label: Dutch
+  prefix: /nl/
+  available: false
```
Comment (KittyGiraudel) [inline _locales/nl.yml:5]: We will need this value to be `true` so Dutch can be displayed in the language picker in the sidebar. :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Read [TRANSLATING.md](TRANSLATING.md) to know more about this.
 
 ## Submitting your work
 
-Once you are done with the translation, make sure your branch is up-to-date with the `master` branch.
+Once you are done with the translation, make sure your branch is up-to-date with the `main` branch.
```
Comment (KittyGiraudel) [inline CONTRIBUTING.md:24]: Good catch, thank you for this. 👍

File: pages/nl/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- _you/your_ vertalen naar _u/uw_ ipv _je/jij/jou_
+- Soms is het handiger en/of klinkt het logischer om een bepaalde uitdrukking in het engels te laten
+
+## Veel voorkomende vertalingen
+
+| engels                         | nederlands                  |
+|--------------------------------|-----------------------------|
+| _declaration_                  | verklaring (of declaratie?) |
+
+## Niet-vertaalbare woorden en uitdrukkingen
```
Comment (KittyGiraudel) [inline pages/nl/README.md:16]: Very good work maintaining that list from the ground up, really happy about it. 😊

File: pages/nl/_extend.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+There are many scenarios where extending selectors are helpful and worthwhile. Always keep in mind these rules so you can `@extend` with care:
+
+* Use extend on `%placeholders` primarily, not on actual selectors.
+* When extending classes, only extend a class with another class, _never_ a [complex selector](https://www.w3.org/TR/selectors4/#syntax).
+* Directly extend a `%placeholder` as few times as possible.
+* Avoid extending general ancestor selectors (e.g. `.foo .bar`) or general sibling selectors (e.g. `.foo ~ .bar`). This is what causes selector explosion.
+
+<div class="note">
+  <p>It is often said that <code>@extend</code> helps with the file size since it combines selectors rather than duplicating properties. That is true, however the difference is negligible once <a href="https://en.wikipedia.org/wiki/Gzip">Gzip</a> has done its compression.</p>
```
Comment (KittyGiraudel) [inline pages/nl/_extend.md:28]: I understand you did not translate this chapter yet, still I wanted to point out we could use the Dutch version of Wikipedia articles when they exist. If you think this is a good idea of course. :)

**Review by KittyGiraudel — COMMENTED** [review]
So I had a first review. Honestly, it’s all great so far, I can see no problem. Feel free to take your time translating the 4 or 5 remaining chapters and then I’ll be more than happy merging it. In the meantime, I listed the Dutch translation with you as a maintainer in #96. :)

**Comment by noah-vdv** [comment]
Thanks!
I'll work on the other translations tomorrow and hopefully thy'll be done by then 👍

**Comment by noah-vdv** [comment]
Alright, all the files are now translated

**Comment by KittyGiraudel** [comment]
@noah-vdv You are a hero. Let me check out the branch and see how it looks like. :)

**Comment by KittyGiraudel** [comment]
Besides 1 broken link, it’s look fantastic so I’m going to merge the PR, and will fix that link on main. :)

**Comment by KittyGiraudel** [comment]
Thank you so much for your hard work @noah-vdv! 💖

**Pull request merged**